### ISSUE TITLE
Train Net Failed ,

### ISSUE THREAD

**Issue by donalwilson** [BVLC/caffe#2863]
the problem is :
I0805 16:30:50.231776 27665 layer_factory.hpp:74] Creating layer data
I0805 16:30:50.231792 27665 net.cpp:90] Creating Layer data
I0805 16:30:50.231798 27665 net.cpp:368] data -> data
I0805 16:30:50.231822 27665 net.cpp:368] data -> label
I0805 16:30:50.231830 27665 net.cpp:120] Setting up data
I0805 16:30:50.231837 27665 data_transformer.cpp:22] Loading mean file from: /home/csursag/caffe-master/examples/Caltech101/Caltech101_mean.binaryproto
I0805 16:30:50.340977 27665 db_leveldb.cpp:17] Opened leveldb /home/csursag/caffe-master/examples/Caltech101/Caltech101_train_leveldb/
I0805 16:30:50.341594 27665 data_layer.cpp:52] output data size: 25,3,227,227
I0805 16:30:50.343574 27665 net.cpp:127] Top shape: 25 3 227 227 (3864675)
I0805 16:30:50.343605 27665 net.cpp:127] Top shape: 25 (25)
I0805 16:30:50.343613 27665 layer_factory.hpp:74] Creating layer conv1
...
...
I0805 16:30:51.719156 27665 net.cpp:192] pool1 needs backward computation.
I0805 16:30:51.719162 27665 net.cpp:192] norm1 needs backward computation.
I0805 16:30:51.719168 27665 net.cpp:192] relu1 needs backward computation.
I0805 16:30:51.719174 27665 net.cpp:192] conv1 needs backward computation.
I0805 16:30:51.719182 27665 net.cpp:194] data does not need backward computation.
I0805 16:30:51.719187 27665 net.cpp:235] This network produces output loss
I0805 16:30:51.719211 27665 net.cpp:482] Collecting Learning Rate and Weight Decay.
I0805 16:30:51.719226 27665 net.cpp:247] Network initialization done.
I0805 16:30:51.719231 27665 net.cpp:248] Memory required for data: 207891304
I0805 16:30:51.719985 27665 solver.cpp:154] Creating test net (#0) specified by net file: /home/csursag/caffe-master/examples/Caltech101/Caltech101_AlexNet_train_val.prototxt
I0805 16:30:51.720057 27665 net.cpp:287] The NetState phase (1) differed from the phase (0) specified by a rule in layer data
I0805 16:30:51.720250 27665 net.cpp:42] Initializing net from parameters: 
name: "AlexNet_UCM"
state {
  phase: TEST
}
...
...
...
I0805 16:30:51.721086 27665 layer_factory.hpp:74] Creating layer data
I0805 16:30:51.721117 27665 net.cpp:90] Creating Layer data
I0805 16:30:51.721124 27665 net.cpp:368] data -> data
I0805 16:30:51.721146 27665 net.cpp:368] data -> label
I0805 16:30:51.721155 27665 net.cpp:120] Setting up data
I0805 16:30:51.721163 27665 data_transformer.cpp:22] Loading mean file from: /home/csursag/caffe-master/examples/Caltech101/Caltech101_mean.binaryproto
I0805 16:30:51.808449 27665 db_leveldb.cpp:17] Opened leveldb /home/csursag/caffe-master/examples/Caltech101/Caltech101_val_leveldb/
*** Aborted at 1438763451 (unix time) try "date -d @1438763451" if you are using GNU date ***
PC: @     0x7f137e2dd974 caffe::Blob<>::Reshape()
*** SIGFPE (@0x7f137e2dd974) received by PID 27665 (TID 0x7f137e97f800) from PID 2116934004; stack trace: ***
    @     0x7f13726eed40 (unknown)
    @     0x7f137e2dd974 caffe::Blob<>::Reshape()
    @     0x7f137e26a124 caffe::DataLayer<>::DataLayerSetUp()
    @     0x7f137e20af4d caffe::BaseDataLayer<>::LayerSetUp()
    @     0x7f137e20b029 caffe::BasePrefetchingDataLayer<>::LayerSetUp()
    @     0x7f137e2c1389 caffe::Net<>::Init()
    @     0x7f137e2c44d1 caffe::Net<>::Net()
    @     0x7f137e2aa3ea caffe::Solver<>::InitTestNets()
    @     0x7f137e2aace7 caffe::Solver<>::Init()
    @     0x7f137e2ab435 caffe::Solver<>::Solver()
    @           0x40d618 caffe::GetSolver<>()
    @           0x407b91 train()
    @           0x404ea8 main
    @     0x7f13726d9ec5 (unknown)
    @           0x404a49 (unknown)
Floating point exception (core dumped)

**Comment by seanbell** [comment]
Can you give steps to reproduce the error?

> When reporting a bug, it's most helpful to provide the following information, where applicable:

> * What steps reproduce the bug?
* Can you reproduce the bug using the latest [master](https://github.com/BVLC/caffe/tree/master), compiled with the `DEBUG` make option?
* What hardware and operating system/distribution are you running?
* If the bug is a crash, provide the backtrace (usually printed by Caffe; always obtainable with `gdb`).

> Try to give your issue a title that is succinct and specific. The devs will rename issues as needed to keep track of them.

**Comment by shelhamer** [comment]
Closing as this looks like a usage issue/request for help.

This tracker is reserved for specific Caffe development issues and bugs; please ask usage questions on the [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) list.

For more information, see our [contributing guide](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md).

Thanks!
